src/evm/contracts/ILightClient.sol/INonMembershipVerifier.ts

```
@@ -0,0 +1,135 @@
+    key: string | FunctionFragment
+  ): T;
+
+  getFunction(
+    nameOrSignature: "verifyNonMembership"
+  ): TypedContractMethod<
+    [proof: Ics23ProofStruct, key: BytesLike],
+    [void],
+    "nonpayable"
+  >;
+
+  filters: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)